src/content/manifest/okteto-variables.mdx

```diff
@@ -43,6 +64,127 @@ To delete an existing variable, click on the **Delete** button on the right. You
   />
 </p>
 
+## Configuring dynamic endpoints
+
+You can create dynamic endpoints for external resources within your Okteto environment using `$OKTETO_ENV`. You can define `$variables`, assign them to an `OKTETO_ENV_VAR`, and make those ENV VARs available in other sections (e.g. `deploy`, `destroy`) of the Oketo manifest to generate dynamic endpoints for external resources. 
```

Comment:
```suggestion
You can create dynamic endpoints for external resources within your Okteto environment using `$OKTETO_ENV`. You can define `$variables`, assign them to `OKTETO_ENV`, and make those variables available in other sections (e.g. `deploy`, `destroy`) of the Okteto manifest or to generate dynamic endpoints for external resources. 
```

Reply:
Making this change locally as part of a batch update. Will push in next commit 👍